lib/rubocop/target_finder.rb

```diff
@@ -55,7 +57,7 @@ def target_files_in_dir(base_dir = Dir.pwd)
       # Support Windows: Backslashes from command-line -> forward slashes
       base_dir = base_dir.gsub(File::ALT_SEPARATOR, File::SEPARATOR) if File::ALT_SEPARATOR
       all_files = find_files(base_dir, File::FNM_DOTMATCH)
-      hidden_files = Set.new(all_files - find_files(base_dir, 0))
+      hidden_files = all_files.select { |file| file.include?(HIDDEN_FILE_PATTERN) }
```

Comment:
I'd add a note we're doing this for performance reasons, as someone might want to optimize it for readability or whatever.

Reply:
adde comment